internal/timecraft/process.go

```diff
@@ -156,6 +162,26 @@ func (pm *ProcessManager) Start(moduleSpec ModuleSpec, logSpec *LogSpec) (Proces
 		}
 	}()
 
+	infc, err := netns.InterfaceByName("en0")
```

Comment:
Should we avoid hard-coding the interface name here? Maybe instead we could loop over interfaces, skip the loopback, and then return the first address we find?

Reply:
I'd say let's keep it as-is for now, I think we'll revisit those soon enough that it's not a big concern 👍